core/src/main/scala/kafka/server/AddPartitionsToTxnManager.scala

```diff
@@ -42,14 +42,26 @@ object AddPartitionsToTxnManager {
   val VerificationTimeMsMetricName = "VerificationTimeMs"
 }
 
+/**
+ * This is an enum which handles the Partition Response based on the Produce Request Version and the exact operation
+ *    defaultOperation:   This is the default workflow which maps to cases when the Produce Request Version was lower than expected or when exercising the offset commit request path
```

Comment:
I think we should include this change in the offset commit path. It will require bumping that request version as well. 

> We will bump the ProduceRequest/Response and TxnOffsetCommitRequest/Response version to indicate the client is using the new protocol that doesn’t require adding partitions to transactions and will implicitly do so. The bump will also support new errors ABORTABLE_ERROR

We are just doing 2 bumps for the above comment

Reply:
It is a similar process to pass in the the value, but we will need it for both group coordinator paths. Let me know if you need help.

Reply:
Done